about.css

```diff
@@ -0,0 +1,434 @@
+  font-size: 3.6vw;
+  font-weight: bold;
+  line-height: 5.7vw;
+}
+
+address {
+  top: -5vw;
+  width: 100%;
+  text-align: center;
+  font-size: 3.6vw;
+  font-weight: 600;
+  text-decoration: underline;
+}
+
+.sectionTitle {
+  font-size: 1.3rem;
+  text-align: center;
+  margin-inline: 3vw;
+  padding-bottom: 5vw;
+  background: url(../capstoneimgs/title_underline.png) bottom no-repeat;
+}
+
+.creative-p {
+  font-size: 0.7rem;
+  line-height: 5vw;
+  font-weight: bold;
+  color: grey;
+  margin-inline: 2vw;
+  text-align: center;
+}
+
+.desktop-p {
+  display: block;
+}
+
+#creative img {
+  width: 50%;
+  margin-block: 3vw;
+  margin-inline: 25vw;
+  padding: 0;
+  border: 2px solid lightgrey;
+}
+
+/******** History ********/
+
+#history {
+  padding: 0 12vw 12vw 12vw;
+}
+
+#history h1 {
+  width: 100%;
+}
+
+.history-p {
+  font-size: 0.7rem;
+  margin-inline: 2vw;
+  font-weight: bold;
+  text-align: center;
+}
+
+.hist-link {
+  width: 30%;
+  height: 30%;
+  color: #fff;
+  text-decoration: none;
+  margin-inline: 1.5vw;
+  transition: transform 300ms ease-in-out;
+}
+
+.hist-link:hover {
+  transform: scale(1.2);
+}
+
+.hist-time {
+  font-size: 9vw;
+  text-align: center;
+}
+
+.hist-title {
+  font-weight: bold;
+  font-size: 3vw;
+  text-align: center;
+}
+
+.ir2022 {
+  width: 100%;
+  display: block;
+  background: url(./capstoneimgs/inti-raymi-2022.jpg) no-repeat;
+  background-size: contain;
+  text-decoration: none;
+  margin-bottom: 1.5vw;
+  color: white;
+}
+
+.ir2020 {
+  width: 100%;
+  display: block;
+  background: url(./capstoneimgs/DSC_01192.jpg) no-repeat;
+  background-size: cover;
+  text-decoration: none;
+  color: white;
+}
+
+#history article {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-color: #b62a1bad;
+  width: 100%;
+  height: 43vw;
+}
+
+#sponsors {
+  position: relative;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  text-align: center;
+  background-color: #414246;
+  padding-bottom: 6vw;
+}
+
+.titlePartners {
+  font-size: 4vw;
+  text-align: center;
+  padding-top: 6vw;
+  padding-bottom: 2vw;
+  color: white;
+  background-color: #414246;
+}
+
+.sponsors {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+/**** Footer ******/
+
+footer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: baseline;
+  background-color: white;
+}
+
+/* .wmf-logo {
+  width: 25vw;
+  position: relative;
+  margin-right: 3vw;
+} */
+
+.footer-p {
+  margin-block: 3vw;
+  color: #414246;
+}
+
+@media screen and (min-width: 768px) {
+  body {
+    overflow-x: hidden;
+  }
+
+  #header-nav-bar {
+    display: none;
+  }
+
+  .session-container {
+    display: flex !important;
+    background-color: black;
+    justify-content: flex-end;
+    align-items: center;
+    color: white;
+    font-size: 14px;
+    height: auto;
+    width: 100%;
+  }
```

Comment:
- Hey, as suggested by the previous reviewer [here](https://github.com/Maedea9/capstone-m1/pull/3/files/1332d8d364cd7085207356e5f74b9a48814a086a#r1142819953), let's a better way of applying styles instead of using the `!important` keyword. please, fix this. they are many that's why he only highlighted one. You should fix this on all occurece in styles except if you're using a CSS framework.

Reply:
Great it is fixed!